modules/stochastic_tools/test/tests/poly_chaos/tests

```diff
@@ -70,4 +70,20 @@
     csvdiff = 'sobol_out_sobol_0002.csv'
     requirement = 'The system shall compute sensitivities sensitivities with computed Legendre polynomial chaos coefficients.'
   []
+  [smolyak_sobol]
+    type = CSVDiff
+    input = sobol.i
+    allow_test_objects = true
+    cli_args = 'Samplers/sample/sparse_grid=smolyak Outputs/file_base=smolyak'
+    csvdiff = 'smolyak_sobol_0002.csv'
+    requirement = 'The system shall use Smolyak sparse grid to integrate polynomials for polynomial chaos expansion.'
+  []
+  [clenshaw_curtis_sobol]
+    type = CSVDiff
+    input = sobol.i
+    allow_test_objects = true
+    cli_args = 'Samplers/sample/sparse_grid=clenshaw-curtis Outputs/file_base=clenshaw_curtis'
+    csvdiff = 'clenshaw_curtis_sobol_0002.csv'
+    requirement = 'The system shall use Clenshaw-Curtis sparse grid to integrate polynomials for polynomial chaos expansion.'
+  []
```

Comment:
```suggestion
[sparse]
  requirement = 'The system shall include the ability to use sparse grid methods to integrate polynomials for polynomical chaos including the'
  [smolyak_sobol]
    type = CSVDiff
    input = sobol.i
    allow_test_objects = true
    cli_args = 'Samplers/sample/sparse_grid=smolyak Outputs/file_base=smolyak'
    csvdiff = 'smolyak_sobol_0002.csv'
    detail = 'Smolyak and'
  []
  [clenshaw_curtis_sobol]
    type = CSVDiff
    input = sobol.i
    allow_test_objects = true
    cli_args = 'Samplers/sample/sparse_grid=clenshaw-curtis Outputs/file_base=clenshaw_curtis'
    csvdiff = 'clenshaw_curtis_sobol_0002.csv'
    detail = 'Clenshaw-Curtis methods.'
  []
```

Reply:
You can create requirement groups for this type of testing. This will need another closing [] for the [sparse] group, I can't figure out how to edit my suggestion.
```suggestion
  [smolyak_sobol]
    type = CSVDiff
    input = sobol.i
    allow_test_objects = true
    cli_args = 'Samplers/sample/sparse_grid=smolyak Outputs/file_base=smolyak'
    csvdiff = 'smolyak_sobol_0002.csv'
    requirement = 'The system shall use Smolyak sparse grid to integrate polynomials for polynomial chaos expansion.'
  []
  [clenshaw_curtis_sobol]
    type = CSVDiff
    input = sobol.i
    allow_test_objects = true
    cli_args = 'Samplers/sample/sparse_grid=clenshaw-curtis Outputs/file_base=clenshaw_curtis'
    csvdiff = 'clenshaw_curtis_sobol_0002.csv'
    requirement = 'The system shall use Clenshaw-Curtis sparse grid to integrate polynomials for polynomial chaos expansion.'
  []
  []
```

Reply:
Maybe I did just edit it. Also, you should use "hit format".

```
cd moose/framework/contrib/hit
make hit
cd to/your/input/file
moose/framework/contrib/hit/hit format -i input.i
```